src/Parser/AfflictionWarlock/Modules/Features/UABuffTracker.js

```diff
@@ -84,7 +84,7 @@ class UABuffTracker extends Module {
   }
 
   statistic() {
-    const buffedTicksPercentage = 1 - (this.unbuffedTicks / this.totalTicks);
+    const buffedTicksPercentage = 1 - (this.unbuffedTicks / (this.totalTicks > 0 ? this.totalTicks : 1));
```

Comment:
Cleaner to do `const unbuffedTicksPercentage = (this.unbuffedTicks / this.totalTicks) || 1;`